### PR TITLE
Create chronograf.subfolder.conf.sample

### PR DESCRIPTION
**Pull request by STaRDoGG**
RP settings for Chronograf: https://www.influxdata.com/time-series-platform/chronograf/

[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

### CODE REVIEW

File: chronograf.subfolder.conf.sample [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    #auth_request /auth;
+    #error_page 401 =200 /ldaplogin;
+
+    # enable for Authelia, also enable authelia-server.conf in the default site config
+    #include /config/nginx/authelia-location.conf;
+
+    include /config/nginx/proxy.conf;
+    resolver 127.0.0.11 valid=30s;
+    set $upstream_app chronograf;
+    set $upstream_port 8888;
```
Comment (Roxedus) [inline chronograf.subfolder.conf.sample:23]: Where does this port come from? All documentation I have found state that it is still port `8086`

Comment (nemchik) [reply chronograf.subfolder.conf.sample:23]: @STaRDoGG can you link to some documentation about the port?

**Review by Erriez — APPROVED** [review]
Please find my comment [286](https://github.com/linuxserver/reverse-proxy-confs/pull/286#discussion_r573905789) on port usage.

My suggestion is to update the following documentation https://docs.linuxserver.io/general/swag:

From:
```
If the proxy_pass statement contains a variable instead of a dns hostname, nginx doesn't check whether it's accessible or not during start. So here we are setting 3 variables:

* **$upstream_app**: Docker container name or external server IP address / hostname.
* **$upstream_port**: Docker container / server port.
* **$upstream_proto**: http or https

The following variables are configured in the example above:
* $upstream_app = heimdall
* $upstream_port = 443
* $upstream_proto = https

`proxy_pass $upstream_proto://$upstream_app:$upstream_port;` results in: `https://heimdall:443`
```
To:
```

```

**Comment by nemchik** [comment]
> Please find my comment [286](https://github.com/linuxserver/reverse-proxy-confs/pull/286#discussion_r573905789) on port usage.
> 
> My suggestion is to update the following documentation https://docs.linuxserver.io/general/swag:
> 
> From:
> 
> ```
> If the proxy_pass statement contains a variable instead of a dns hostname, nginx doesn't check whether it's accessible or not during start. So here we are setting 3 variables:
> 
> * **$upstream_app**: Docker container name or external server IP address / hostname.
> * **$upstream_port**: Docker container / server port.
> * **$upstream_proto**: http or https
> 
> The following variables are configured in the example above:
> * $upstream_app = heimdall
> * $upstream_port = 443
> * $upstream_proto = https
> 
> `proxy_pass $upstream_proto://$upstream_app:$upstream_port;` results in: `https://heimdall:443`
> ```
> 
> To:
> 
> ```
> 
> ```

Are you saying to remove that section? Or did your recommendation for what to put not get posted?

**Review by nemchik — APPROVED** [review]

**Pull request merged**